app/packages/core/src/components/utils/editor/klogs.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable sort-keys */
+import { languages } from 'monaco-editor';
```

Comment:
This has nothing to do with laziness, but with the readability of the file.
Sometimes I want a property called `closed` to be after the property `open`, because it makes more sense compared to the lexical ordering. I will reformat the file, because I don't have the energy to argue this.